lib/vmdb/plugins.rb

```diff
@@ -100,16 +100,20 @@ def register_models
     # Determine the version of the specified engine
     #
     # If the gem is
-    # - git based, pointing to a branch:   <branch>@<sha>
-    # - git based, pointing to a tag:      <tag>@<sha>
-    # - git based, pointing to a sha:      <sha>
-    # - path based, with git, on a branch: <branch>@<sha>
-    # - path based, with git, on a tag:    <tag>@<sha>
-    # - path based, with git, on a sha:    <sha>
-    # - path based, without git:           nil
-    # - a real gem:                        <gem_version>
+    # - git based, pointing to a branch:             <branch>@<sha>
+    # - git based, pointing to a tag:                <tag>@<sha>
+    # - git based, pointing to a sha:                <sha>
+    # - path based, with git, on a branch:           <branch>@<sha>
+    # - path based, with git, on a tag:              <tag>@<sha>
+    # - path based, with git, on a sha:              <sha>
+    # - path based, without git:                     nil
+    # - symlinked path based, with git, on a branch: <branch>@<sha>
+    # - symlinked path based, with git, on a tag:    <tag>@<sha>
+    # - symlinked path based, with git, on a sha:    <sha>
+    # - symlinked path based, without git:           nil
```

Comment:
Is there really a point of adding these extra comments?  I think it just adds to the confusion since the syntax is still the same.  Maybe instead just say "path can also be a symlink"?